x-pack/solutions/observability/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -317,11 +318,16 @@ export const createInventoryMetricThresholdExecutor =
           ...additionalContext,
         };
 
+        const {
```

Comment:
The `libs.sources.getSourceConfiguration` is already used in this file. Just curious, is it not possible to call it only once and use `metricAlias` from there?